man/runc-delete.8.md

```diff
@@ -1,11 +1,14 @@
 # NAME
-   runc delete - delete any resources held by the container often used with detached containers
+   runc delete - delete any resources held by one or more containers often used with detached containers
 
 # SYNOPSIS
-   runc delete <container-id>
+   runc delete <container-id> [container-id...]
```

Comment:
thanks, I change it to `runc delete  [command options] [container-id...] <container-id>`, because `runc delete`  requires a minimum of one argument.